functions-and-operators/string-functions.md

```diff
@@ -156,10 +156,92 @@ Decode to a base-64 string and return result.
 ### [`HEX()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_hex)
 
 Return a hexadecimal representation of a decimal or string value.
+- For a string argument `HEX(str)` returns a hexademical string representation of `str` where each byte of each character in `str` is converted to two hexadecimal digits.
+- The inverse of this operation is performed by the [ `UNHEX()` ](https://docs.pingcap.com/tidb/dev/string-functions#unhex) function.
+- For a numeric argument `HEX(n)` returns a hexadecimal string representation of the value of `n` treated as a longlong (`BIGINT`) number. This is equivalent to [ `CONV(n, 10, 16)` ](https://docs.pingcap.com/tidb/stable/numeric-functions-and-operators#mathematical-functions).
+- For a `NULL` argument, this function returns `NULL`.
+
+```sql
+SELECT X'616263', HEX('abc'), UNHEX(HEX('abc'));
```

Comment:
Maybe add `0x616263`?
```
mysql> SELECT X'616263', HEX('abc'), UNHEX(HEX('abc')), 0x616263;
+-----------+------------+-------------------+----------+
| X'616263' | HEX('abc') | UNHEX(HEX('abc')) | 0x616263 |
+-----------+------------+-------------------+----------+
| abc       | 616263     | abc               | abc      |
+-----------+------------+-------------------+----------+
1 row in set (0.00 sec)
```

Reply:
Nice!
I add it as you said > https://github.com/pingcap/docs/pull/16114/commits/f4d74302a4857828117439221e4a7d38e264d632